packages/sdk/explorer/src/block-explorer.ts

```diff
@@ -315,8 +315,16 @@ export class BlockExplorer {
   getContractMappingFromSourcify = async (
     address: string
   ): Promise<ContractMapping | undefined> => {
+    const cached = this.addressMapping.get(address)
```

Comment:
TODO would there be a reason why mappings from sourcify should not be cached in the way core contracts are?

Reply:
🤷